kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -1,263 +1,384 @@
 set -e
 
-echo "Starting deploying Kubernetes cluster"
-date
-
-echo "Running as:"
-whoami
+### 
+#	<summary>
+#   	Logs output by prepending date and log output type(Error, warning, info or verbose).
```

Comment:
I think the term "log level"  in more standard than "output type"

Reply:
Changed to log level.